com.unity.render-pipelines.high-definition/Editor/Wizard/HDWizard.Window.cs

```diff
@@ -137,18 +138,30 @@ public ConfigStyle(string label, string error, string button = resolve, MessageT
                 label: "Direct3D 12",
                 error: "Direct3D 12 is needed! (Editor restart is required)");
             public static readonly ConfigStyle dxrScreenSpaceShadow = new ConfigStyle(
-                label: "Screen Space Shadow",
-                error: "Screen Space Shadows are disabled in the current HDRP asset. You will not be able to toggle ray traced shadows on the lights in your scene. You can enable the feature in the HDRP asset under Lighting -> Shadows -> Screen Space Shadows", messageType: MessageType.Info);
+                label: "Screen Space Shadows (Asset)",
+                error: "Screen Space Shadows are disabled in the current HDRP asset. You will not be able to toggle ray traced shadows on the lights in your scene. You can enable the feature in the HDRP asset under Lighting > Shadows > Screen Space Shadows", messageType: MessageType.Warning);
+            public static readonly ConfigStyle dxrScreenSpaceShadowFS = new ConfigStyle(
+                label: "Screen Space Shadows (Default Camera Frame Setting)",
+                error: "Screen Space Shadows are disabled in the default Camera Frame setting. You can enable the feature in Project Settings > HDRP Default Settings > Frame Settings > Default Frame Settings For Camera > Lighting > Screen Space Shadows", messageType: MessageType.Info);
             public static readonly ConfigStyle dxrReflections = new ConfigStyle(
-                label: "Reflections",
-                error: "Screen Space Reflections are disabled in the current HDRP asset. You will not be able to toggle ray traced reflections though your volume components. You can enable the feature in the HDRP asset under Lighting -> Reflections -> Screen Space Reflections", messageType: MessageType.Info);
+                label: "Reflection (Asset)",
+                error: "Screen Space Reflection is disabled in the current HDRP asset. You will not be able to toggle ray traced reflections though your volume components. You can enable the feature in the HDRP asset under Lighting > Reflections > Screen Space Reflections", messageType: MessageType.Warning);
```

Comment:
Screen Space Reflection is disabled in the current HDRP Asset which means you cannot enable ray-traced reflections in Volume components. To enable this feature, open your HDRP Asset, go to **Lighting > Reflections**, and enable **Screen Space Reflections**.